src/wallet/home_cards/manager.rs

```diff
@@ -474,4 +486,27 @@ mod tests {
         let handled_cards = observer.handled_cards.lock().unwrap().clone();
         pretty_assertions::assert_eq!(handled_cards.unwrap(), initial_cards);
     }
+
+    #[actix_rt::test]
+    async fn test_wallet_reset() {
+        let initial_cards = HomeCards::from_iter(vec![
+            HomeCard::ContinueRadQuest,
+            HomeCard::Connector,
+        ]);
+        let observer = Arc::new(MockHomeCardsObserver::new());
+        let manager = SUT::new(
+            Arc::new(MockAntenna::new_always_failing()),
+            NetworkID::Stokenet,
+            Arc::new(MockHomeCardsStorage::new_with_stored_cards(
+                initial_cards.clone(),
+            )),
+            observer.clone(),
+        );
+
+        manager.bootstrap().await.unwrap();
```

Comment:
I think we should remove this `bootstrap()` call to ensure the modification takes place on `wallet_resetted`

Reply:
But then the test wouldn't be complete as the manager wouldn't have any cards in memory. The test will pass even if the wallet_reset doesn't do anything, but calling `bootstrap()` loads the mocked cards into the `HomeCardsManager` memory so that the test makes sense.

Reply:
oh, I thought the test has the `initial_cards` initally set